libs/deps/merch-icon.js

```diff
@@ -0,0 +1,29 @@
+// branch: develop commit: 369516f3cda51fb1219ad0b3cf2c94c8f094c49b Tue, 21 May 2024 08:39:16 GMT
```

Comment:
I see a lot of merch related files being added to `/deps`, maybe it's time to create a `/deps/merch` folder and host everything in there.

Reply:
actually, after this PR we wanted to discuss the possibility of moving our web components back to milo. 
But even in that case we could have a separate folder, yes.
If it's ok with you @narcis-radu I'd move them in the separate PR, as this one has already so many changes